railties/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+
+*   Expose `Rack::Request::Helpers#request_method` on `ActionDispatch::Request` so that `Rails::Rack::Logger#started_request_message` can call it without raising `ActionController::UnknownHttpMethod`, which isn't handled correctly at this point in the middleware stack.
```

Comment:
This covers the implementation, but the user doesn't need to know any of this to get the benefit of the change.

```suggestion
*   Return a 405 Method Not Allowed response when a request uses an unknown HTTP method.
```

Basically the same changelog entry as https://github.com/rails/rails/pull/10299, but in a different framework 😄